### PR TITLE
poc: get icons working with relative import

### PR DESCRIPTION
**Pull request by micmro**
Correct import will require a more elaborate setup to add TS compitlation to foundations.

I also noticed the NX setup does not look quite standard as what you'd get using the CLI.

```bash
npx create-nx-workspace@latest nx-radius-reference-package-based --preset=ts

# go into monrepo
cd nx-radius-reference-package-based

# add libraries
npx nx generate @nx/js:library foundation --publishable --importPath @rangle/radius-foundations
npx nx generate @nx/js:library core-components --publishable --importPath @rangle/radius-react-core-components


# addd demo app
npm install --save-dev @nx/remix
npx nx g @nx/remix:setup
npx nx g @nx/remix:app demo
```

### CODE REVIEW

File: apps/demo/app/routes/index.tsx [hunk 1]
```diff
@@ -95,9 +101,9 @@ export default function Index() {
             header: 'Header',
             headingLevel: 'h2',
             src: 'https://i.ibb.co/MS0V7Zk/elisa-pitkanen-Z9-FZ6hko-k-unsplash.jpg',
-            ['data-radius-watch-token-changes']: true,
-            ['data-radius-token']:
-              '--asset-component-image-large-image-text-item-image01',
+            // ['data-radius-watch-token-changes']: true,
+            // ['data-radius-token']:
+            //   '--asset-component-image-large-image-text-item-image01',
```
Comment (micmro) [inline apps/demo/app/routes/index.tsx:106]: Are these needed?

File: apps/demo/app/routes/index.tsx [hunk 2]
```diff
@@ -1,34 +1,39 @@
...
-// @ts-nocheck - This needs to be rebuilt
-
-import React from 'react';
+import React, { ComponentType, SVGProps } from 'react';
+import Github from '../../../../library/foundations/generated/icons/github';
+import Figma from '../../../../library/foundations/generated/icons/figma';
+import EventNote from '../../../../library/foundations/generated/icons/event-note';
+// import Github from '@rangle/radius-foundations/generated/icons/github.tsx';
+// import Figma from '@rangle/radius-foundations/generated/icons/figma';
+// import EventNote from '@rangle/radius-foundations/generated/icons/event-note';
```
Comment (micmro) [inline apps/demo/app/routes/index.tsx:7]: To import them correctly we'll need to change to the `@rangle/radius-foundations` which will take a while.

File: apps/demo/app/routes/index.tsx (hunk 1 above)
Comment (nkanigsberg) [reply apps/demo/app/routes/index.tsx:106]: I don't think so no. Seems like a debugging artifact

Comment (nkanigsberg) [reply apps/demo/app/routes/index.tsx:106]: Actually, this might be what @jasonsantos  was using to watch for changes to context?

**Pull request merged**